starter-code/src/viking.js

```diff
@@ -1,8 +1,47 @@
 // Soldier
-function Soldier() {}
+class Soldier {
+  constructor (healthArg, strengthArg) {
+    this.health = healthArg
+    this.strength = strengthArg
+  }
+
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(damage) {
+    this.health = this.health - damage 
+  } 
+}
+
 
 // Viking
-function Viking() {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(name, health, strength);
```

Comment:
You could have simplified this a little 

``` 
  constructor(name, health, strength) {
    super(health, strength);
    this.name = name;
  } 
```

The health and strength is getting passed from the Solider function